docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "2"
+services:
+  web:
+    name: "yacy"
+    build: .
+    ports:
+      - "80:80"
+    links:
+      - elasticsearch
+    environment:
+      ELASTIC_URI: elasticsearch://localhost:5050/yacy_grid_mcp
+
+  elasticsearch:
+    image: elasticsearch
```

Comment:
replace this with compose format given at https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html